backup-tools/repl-image/files/replication-run.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
-chmod 600 /backup/env/*
-cron -f
+while true; do
+  /usr/local/sbin/backup-replication.sh
+  sleep 14400
```

Comment:
It's only the replication image that gets changed here, for the backup image there will be another pull request in a few minutes.